app/models/audit/event/fetch.rb

```diff
@@ -28,6 +30,10 @@ def progname
         Event.progname
       end
 
+      def action_sd
```

Comment:
@liubovra what does `sd` stand for? can we add a comment?

Reply:
the action_sd is existing field in attempted_action.action_sd.
I just added a getter for it so I can add a unit test for checking the operation field as there was no unit test for it.

Reply:
still would be nice to know what it stands for

Reply:
It's like "structured data" (see https://tools.ietf.org/html/rfc5424)

Reply:
@orenbm Added a comment